scripts/Imaging/imgpy/filters/remove_sino_stripes_rings_wf.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+
+def remove_sino_stripes_rings_wf(data_vol, wv_levels=None):
+    if not wv_levels:
+        max_len = np.max(data_vol.shape)
+        wv_levels = int(np.ceil(np.log2(max_len)))
+
+    return _remove_sino_stripes_rings_wf(data_vol, wv_levels)
+
+
+# The code of the 'Wavelet-Fourier' stripe removal method is heavily based on
+# the implementation of this method in tomopy
+# The original code from Tomopy is:
```

Comment:
To meet the copyright requirements you need to say if we have modified any of the code here from `Tomopy` and if so what? 

Additionally a link back to the Tomopy source file is always good here